src/content/docs/fr/guides/deploy/clever-cloud.mdx

```diff
@@ -10,42 +12,22 @@ import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
 
 ## Configuration du projet
 
-Vous pouvez déployer à la fois des projets Astro entièrement statiques ou rendus à la demande sur Clever Cloud. Quel que soit votre mode de sortie ([pré-rendu ou à la demande](/fr/guides/on-demand-rendering/)), vous pouvez choisir de déployer en tant qu'**application statique** qui s'exécute à l'aide d'un hook de post-construction, ou en tant qu'application **Node.js**, qui fonctionne immédiatement avec votre fichier `package.json`.
+Vous pouvez déployer à la fois des projets Astro entièrement statiques ou rendus à la demande sur Clever Cloud. Quel que soit votre mode de sortie ([pré-rendu ou à la demande](/fr/guides/on-demand-rendering/)), vous pouvez choisir de déployer en tant qu'**application statique** qui s'exécute à l'aide d'un hook de post-construction, ou en tant qu'application **Node.js**, ce qui nécessite une configuration manuelle dans votre `package.json`.
 
-### Port et hôte
-
-Les applications sur Clever Cloud écoutent sur le port **8080**. Si votre projet nécessite cette configuration, définissez votre port et votre hôte dans Astro à l'un des deux emplacements suivants :
-
-1. Dans les scripts `package.json` :
-
-    ```json title="package.json" "astro preview --host 0.0.0.0 --port 8080"
-    "scripts": {
-      "dev": "astro dev",
-      "start": "astro dev",
-      "build": "astro check && astro build",
-      "preview": "astro preview --host 0.0.0.0 --port 8080",
-      "astro": "astro"
-    } 
-    ```
-
-2. Dans `astro.config.mjs` :
-
-    ```javascript title="astro.config.mjs"
-    import { defineConfig } from 'astro/config';
-
-    export default defineConfig({
-      server: {
-        port: 8080,
-        host: true
-      }
-    });
-    ```
+### Scripts
 
+Si vous exécutez une application Node.js à la demande, mettez à jour votre script `start` pour exécuter le serveur Node. Les applications sur Clever Cloud écoutent sur le port **8080**.
```

Comment:
I think you put the code snippet in the wrong place. I believe it should be here:
````suggestion
Si vous exécutez une application Node.js à la demande, mettez à jour votre script `start` pour exécuter le serveur Node. Les applications sur Clever Cloud écoutent sur le port **8080**.

```json title="package.json"
"scripts": {
  "start": "node ./dist/server/entry.mjs --host 0.0.0.0 --port 8080",
} 
```
````